Source/Krypton Components/Krypton.Toolkit/General/KryptonAboutToolkitData.cs

```diff
@@ -226,57 +226,57 @@ public KryptonAboutToolkitData(ToolkitType toolkitType = ToolkitType.Stable)
                                        LinkArea? learnMoreLinkArea, LinkArea? discordLinkArea, LinkArea? repositoryInformationLinkArea,
                                        LinkArea? downloadDemosLinkArea, LinkArea? documentationLinkArea)
```

Comment:
The idea odf the struct, was to replace all of these Parameters being passed in here !!

Reply:
Ok, so now there is one public function to call:

```cs
public static DialogResult Show(KryptonAboutToolkitData data)
            => ShowCore(data.HeaderText, data.GeneralInformationWelcomeText, data.GeneralInformationLicenseText,
                        data.GeneralInformationLearnMoreText, data.CurrentThemeText, data.DiscordText,
                        data.RepositoryInformationText, data.DownloadDocumentationText, data.DownloadDemosText,
                        data.FileNameColumnHeaderText, data.VersionColumnHeaderText, data.ToolBarGeneralInformationText,
                        data.ToolBarDiscordText, data.ToolBarDeveloperInformationText, data.ToolBarVersionInformationText,
                        data.LearnMoreLinkArea, data.DiscordLinkArea, data.RepositoryInformationLinkArea,
                        data.DownloadDemosLinkArea, data.DocumentationLinkArea, data.ToolkitType,
                        data.ShowDiscordButton, data.ShowDeveloperInformationButton,
                        data.ShowVersionInformationButton,
                        data.ShowThemeOptions, data.ShowSystemInformationButton);
```

Reply:
And use it in the ShowCore as well !

Reply:
> And use it in the ShowCore as well !

Been revised since, so it's now:

```cs
/// <summary>Shows a new <see cref="KryptonAboutToolkitForm"/>.</summary>
        /// <param name="aboutToolkitData">The data to pass through.</param>
        /// <returns>A new <see cref="KryptonAboutToolkitForm"/> with the specified data.</returns>
        public static DialogResult Show(KryptonAboutToolkitData aboutToolkitData)
            => ShowCore(aboutToolkitData);
```

and

```cs
private static DialogResult ShowCore(KryptonAboutToolkitData aboutToolkitData)
        {
            using var kat = new KryptonAboutToolkitForm(aboutToolkitData);

            return kat.ShowDialog();
        }
```

which calls:

```cs
 public KryptonAboutToolkitForm(KryptonAboutToolkitData aboutToolkitData)
        {
            InitializeComponent();

            InitialiseDialog(aboutToolkitData);

            kbtnOk.Text = KryptonManager.Strings.GeneralStrings.OK;

            kbtnSystemInformation.Text = KryptonManager.Strings.CustomStrings.SystemInformation;
        }
```

Reply:
I've added a demo to show how to use it